packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -56,15 +56,6 @@ typedef _DismissCallback = void Function(
   double opacity,
 );
 
-/// A function that produces the preview when the CupertinoContextMenu is open.
-///
-/// Called every time the animation value changes.
-typedef ContextMenuPreviewBuilder = Widget Function(
```

Comment:
This wasn't deprecated in the deprecation PR but it doesn't seem to be used anywhere else.